### PR TITLE
Hotfix/Voting systems

### PR DESCRIPTION
**Pull request by VirginiaDooley**
https://trello.com/c/mv68Ztw7/2713-add-stv-explainer-to-wcivf

- Translates the How to Vote template which was left out of previous translations. 
- Adds the STV voting system template including translations

### CODE REVIEW

File: wcivf/apps/elections/urls.py [hunk 1]
```diff
@@ -74,4 +74,12 @@
         ),
         name="sv_voting_system_view",
     ),
+    re_path(
+        r"^voting_system/stv/",
+        TranslatedTemplateView.as_view(
```
Comment (michaeljcollinsuk) [inline wcivf/apps/elections/urls.py:79]: Do you need to rerun `makemessages`? Or is it not necessary for `TranslatedTemplateView`?

Comment (symroe) [reply wcivf/apps/elections/urls.py:79]: No, for this we just email the HTML file to the translators and paste their response in a `filename_[language].html` file

File: wcivf/apps/elections/templates/elections/includes/_how-to-vote.html [hunk 2]
```diff
@@ -1,61 +1,105 @@
...
 <ul class="ds-details">
     <li style="list-style: none;">
         <details>
-            <summary>How you vote</summary>
+            <summary>{% trans 'How you vote' %}</summary>
             <p>
                 {% if voting_system.get_absolute_url %}
-                    This election uses <a href="{{ voting_system.get_absolute_url }}">{{ voting_system.name }}</a>.
+                    {% blocktrans trimmed with voting_system_url=voting_system.get_absolute_url %}
+                        This election uses <a href="{{ voting_system_url }}">{{ voting_system.name }}</a>.
```
Comment (symroe) [inline wcivf/apps/elections/templates/elections/includes/_how-to-vote.html:21]: `voting_system.name` will fail here I think?

File: wcivf/apps/elections/templates/elections/includes/_how-to-vote.html [hunk 3]
```diff
@@ -1,61 +1,105 @@
...
+                    {% endblocktrans %}
                 {% else %}
-                    This election uses {{ voting_system.name }}.
+                    {% blocktrans trimmed with voting_system_name=voting_system.name %}
+                        This election uses {{ voting_system.name }}.
```
Comment (symroe) [inline wcivf/apps/elections/templates/elections/includes/_how-to-vote.html:25]: And here

File: wcivf/apps/elections/templates/elections/includes/_how-to-vote.html (hunk 2 above)
Comment (VirginiaDooley) [reply wcivf/apps/elections/templates/elections/includes/_how-to-vote.html]: Fixed in last fixup commit.

File: wcivf/apps/elections/templates/elections/includes/_how-to-vote.html (hunk 3 above)
Comment (VirginiaDooley) [reply wcivf/apps/elections/templates/elections/includes/_how-to-vote.html]: Fixed [here](https://github.com/DemocracyClub/WhoCanIVoteFor/pull/1203/commits/c0d2f86c68959958fe350b193dbb05456c6a1b3c)

**Review by symroe — APPROVED** [review]

**Pull request merged**